packages/i18n/src/translateWithMarkup.js

```diff
@@ -120,8 +120,10 @@ function TranslateWithMarkup({ mapping = {}, children }) {
     );
   }
 
-  const node = new DOMParser().parseFromString(children, 'text/html').body
-    .firstChild;
+  const node =
+    typeof window !== 'undefined'
+      ? new DOMParser().parseFromString(children, 'text/html').body.firstChild
```

Comment:
Ah, forgot about that.

Does this change break anything in `TranslateWithMarkup`?

Wonder if some Jest unit tests with `@jest-environment node` would be useful here as well.